### DISCUSSION TITLE
How create a new unpacker plugin?

### DISCUSSION THREAD

**Discussion by GianlucaCr** [WinMerge/winmerge#1429] [Ideas]
I need an unpacker plugin for Visual FoxPro files. The decompressor is https://github.com/fdbozzo/foxbin2prg but I don't know how to create a plugin. Is there any online tutorial explaining how to do it? Or even better, is there anyone willing who would do it to make it available to everyone?

**Comment by sdottaka** [comment]
Unfortunately, there is no tutorial documentation available.
It has long since been updated, but here is the documentation for the plugin
https://github.com/WinMerge/winmerge/blob/master/Plugins/README.md
Also, sample plug-ins are stored in the following folder
https://github.com/WinMerge/winmerge/tree/master/Plugins
